internal/host/plugin/job_set_sync.go

```diff
@@ -466,7 +480,7 @@ func (r *SetSyncJob) upsertHosts(
 							return err
```

Comment:
Does this code path need test coverage?

Reply:
Should be tested via https://github.com/hashicorp/boundary/pull/1707/files/ddadd262d8c89e309a1dc7ca83b8088fac5b450f

Originally the upsert logic was in the repo but Todd pulled them into the job, so the test functions are a bit split...

Reply:
ty!